src/views/SpaceSettings.vue

```diff
@@ -113,6 +113,8 @@ function pluginName(key) {
 async function handleSubmit() {
   if (isValid.value) {
     if (form.value.filters.invalids) delete form.value.filters.invalids;
+    if (!form.value.voting.delay) delete form.value.voting.delay;
```

Comment:
This feels a bit weird when reading. 

It reads like if there is no(falsy) voting delay, delete it. 

Why do we need to delete it?